lib/src/components/NxToast/NxToast.scss

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-toast__wrapper {
+  ///nx-toast__wrapper has position absolute to ensure toasts stay in the top right corner of the page.
+  //Using display:flex and flex-end ensures the right side of the toast is aligned with the right side of 
+  //nx-page-main. The nx-toast_container has position fixed applied so it remains fixed while scrolling.
+  position: absolute;
+  top: 0;
+  right: 0;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: var(--nx-spacing-6x);
+}
+
+.nx-toast__container {
+  position: fixed;
+  width: 500px;
+}
+
+.nx-toast {
+  position: relative;
+  animation: animationIn 0.4s linear;
+
+  &.nx-toast--closing {
+    animation: animationOut 0.4s linear;
+  }
+  
+  .nx-alert {
+    margin: var(--nx-spacing-2x) 0 0 0;
+  }
+}
+
+@keyframes animationIn {
+  0% {
+    margin-top: 0;
+    max-height: 0px;
+    right: -500px;
+    opacity: 0;
+  }
+  40% {
+    margin-top: var(--nx-spacing-2x);
+    right: -500px;
+    opacity: 0;
+    max-height:74px;
+  }
+  100% {
+    right:0;
+    opacity: 1;
+  }
+}
+
+@keyframes animationOut {
+  0% {
+    max-height: 75px;
+    right: 0;
+    opacity: 1;
+  }
+  75% {
+    max-height:74px;
```

Comment:
Hmm so it's 74 here but 75 at 0%? Does that do anything? I'd expect not since 74 is the height of the content

Reply:
> Hmm so it's 74 here but 75 at 0%? Does that do anything? I'd expect not since 74 is the actual height of the content

Nope, it didn't do anything, and was a typo. Fixed here when removing initial state for animationOut: [ca49163](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/ca49163b4dce7eee38131b2d3ca3b6652eff2c44)